iaf-commons/src/main/java/nl/nn/adapterframework/util/StringUtil.java

```diff
@@ -283,4 +287,36 @@ public static Stream<String> splitToStream(@Nullable final String input, @Nonnul
 		return splitPattern.splitAsStream(input.trim())
 				.filter(StringUtils::isNotBlank);
 	}
+
+	/**
+	 * toStrings and concatenates all fields of the given object, except fields containing the word 'password'
+	 * Uses the {@link ToStringStyle#DEFAULT_STYLE DEFAULT_STYLE} by default.
+	 * 
+	 * @see org.apache.commons.lang3.builder.ToStringBuilder#reflectionToString
+	 */
+	@Nonnull
+	public static String reflectionToString(Object object) {
+		return reflectionToString(object, ToStringStyle.DEFAULT_STYLE);
+	}
+
+	/**
+	 * toStrings and concatenates all fields of the given object, except fields containing the word 'password'
+	 * @param style the ToStringStyle to use.
+	 * 
+	 * @see org.apache.commons.lang3.builder.ToStringBuilder#reflectionToString
+	 */
+	@Nonnull
+	public static String reflectionToString(Object object, ToStringStyle style) {
+		try {
+			return (new ReflectionToStringBuilder(object, style) {
+				@Override
+				protected boolean accept(Field f) {
+					return super.accept(f) && !f.getName().contains("password");
```

Comment:
1. Ik zou dit case-insensitive willen doen (er is bv ook een veld `wssPassword` ergens, nu geen issue maar zou het kunnen worden)
2. Wat voor andere fieldnames zouden gevoelige informatie kunnen bevatten zoals auth tokens, client secrets?

Die velden zitten nu niet in classes die een `toString()` hebben maar ik probeer een beetje vooruit te denken.

Verder lijkt dit mij de juiste richting om in te gaan.